src/main/java/org/jenkinsci/maven/plugins/hpi/MavenArtifact.java

```diff
@@ -244,9 +245,9 @@ public String getActualVersion() throws IOException {
         }
     }
 
-    public ArtifactVersion getVersionNumber() throws OverConstrainedVersionException {
-        return artifact.getSelectedVersion();
-    }
+    // public ArtifactVersion getVersionNumber() throws OverConstrainedVersionException {
+    //     return artifact.getSelectedVersion();
+    // }
```

Comment:
This changes the public API by removing the `getVersionNumber` method.  Have you confirmed there are no consumers outside this repository that depend on the public API?

If so, then I believe the comment block should be deleted.
```suggestion
```